BlogTemplate/Pages/Drafts.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿@page
+@model DraftsModel
+@{
+    ViewData["Title"] = "Drafts";
+}
+<h2>@ViewData["Title"]</h2>
+
+<p>This is where you go to edit drafts.</p>
+
+@{
+    foreach (var post in Model.Blog.Posts)
+    {
+        if (!post.IsPublic)
+        {
+            <article>
+                <h2><a href="/post/@post.Slug">@post.Title</a></h2>
```

Comment:
The title of the draft should link to its Edit page rather than to its Post page. That way when the user clicks on the title they can immediately start editing the draft rather than going to the Post page and clicking on "Edit Post".